runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java

```diff
@@ -1589,4 +1590,16 @@ public void clearClientCertPreferences(Runnable callback) {
         checkThreadSafety();
         mContent.clearClientCertPreferences(callback);
     }
+
+    /**
+     * Gets the SSL certificate for the main top-level page or null
+     * if there is no certificate (the site is not secure).
```

Comment:
is it better to add "* @return the SSL certificate for the main top-level page " here?